pkg/csi/service/wcp/controller.go

```diff
@@ -148,6 +147,19 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 						time.Sleep(5 * time.Second)
 					}
 				}
+				if event.Op&fsnotify.Create == fsnotify.Create {
+					if event.Name == cnsconfig.SupervisorCAFilePath {
+						for {
```

Comment:
We should print the log message about certificate is rotated, before the for a loop.

Reply:
Done